### ISSUE TITLE
[ktlint 0.37.0] New indentation rule issues 

### ISSUE THREAD

**Issue by gotev** [pinterest/ktlint#764]
## Expected Behavior
When I have statements like those:

```kotlin
myStatement(parameter1, parameter2, parameter3)

myOtherStatement(parameter1, lambda = {
    otherStatement
})
```

I expect the linter to report nothing

## Observed Behavior
I get a bunch of:

```
Missing newline after "("
Missing newline after ","
Missing newline before ")"
```

## Your Environment
* Version of ktlint used: 0.37.0
* Operating System and version: macOS Catalina 10.15.5

With ktlint up to `0.36.0` no problems. I believe it's something with the new indentation rule promoted from experimental. Is it possible to keep the "old" behaviour? I find it excessive to have very short statements split on multiple lines only for a `ktlint` rule.

**Comment by shashachu** [comment]
Hi, yes the formerly-experimental-now-standard rule is more aggressive with inserting newlines, and I understand IDEA also accepts the formatting you've shown in the issue. We'll discuss internally whether any changes need to be made.

**Comment by Tapchicoma** [comment]
After some discussion we decided to split indentation rule into more granular rules, that could be disabled via `.editorconfig`. Default behaviour would be as observed one.

**Comment by douglaskazumi** [comment]
@Tapchicoma what's the correct rule to be disabled in this case? Is it `indent` or is there any more specific rule I can disable?

Thanks

**Comment by romtsn** [comment]
Yes, it is the entire `indent` rule at the moment

**Comment by Kim1Jun** [comment]
I think that `indent` rule is too comprehensive.
How about add specific rule `aggressive-inserting-newlines` to disable issue's example only?

**Comment by realdadfish** [comment]
Yes, please, this currently stops us from upgrading to 0.37 or later, otherwise we'd face thousands of lines of reformatting :(

**Comment by kharf** [comment]
same here, we don't want the curly braces to be placed in new lines!

**Comment by maxirosson** [comment]
Any news about this?

**Comment by firmwaves** [comment]
@Tapchicoma I'm guessing the 0.41.0 milestone is incorrect since it has been released? any other updates here?
